docs/buildah-bud.md

```diff
@@ -327,6 +331,10 @@ another process.
 
 Do not use existing cached images for the container build. Build from the start with a new set of cached layers.
 
+**--os**="OS"
+
+Prefer OS instead of the running OS when building images.
```

Comment:
Should this be `Set OS ...` as well.

Reply:
Should we default to the current OS, so that the `buildah bud --help` shows the default?

Reply:
Fixed that, the help shows the system default values:
```
 > ./buildah bud -h | rg '\--(arch|os|platform)'
      --arch string                   set the ARCH instead of the architecture of the current machine in the image manifest and config (default "amd64")
      --os string                     set the OS instead of the current operating system of the machine in the image manifest and config (default "linux")
      --platform linux/arm            set the OS/ARCH instead of the current operating system and architecture of the machine in the image manifest and config (for example linux/arm) (default "linux/amd64")
```